tests/Doctrine/Migrations/Tests/Tools/TransactionHelperTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Tests\Tools;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\Connection as DriverConnection;
+use Doctrine\DBAL\Driver\PDO\Connection as PDOConnection;
+use Doctrine\Migrations\Tools\TransactionHelper;
+use PHPUnit\Framework\TestCase;
+
+final class TransactionHelperTest extends TestCase
+{
+    /**
+     * @param class-string $driverConnectionFqcn
```

Comment:
```suggestion
     * @param class-string<DriverConnection> $driverConnectionFqcn
```

Reply:
I cannot do this because that interface does not have the `inTransaction()` method and therefore PHPStan complains that PHPUnit cannot mock it